tests/test_install.sh

```diff
@@ -17,50 +17,32 @@ rm -rf "$working_dir"
 mkdir -p "$working_dir"
 cd "$working_dir"
 
+# Start ephemeral network in the background
+network_live_time=60
+timeout --signal=SIGINT --kill-after=${network_live_time}s --preserve-status ${network_live_time}s \
+"$INSTALL_PREFIX"/bin/sandbox.sh --verbose &
+
+# Wait for service to be open
+sleep 45
```

Comment:
That seems like a long time, but I guess we don't have a convenient file to wait on anymore.

Reply:
Yes I think we should find a better way to wait. We could grep the output of `sandbox.sh`, or wait for some file to be written by the infra, or have a separate util (in bash or Python) which polls the node until it seems active?

Reply:
Good point. I've added a quick check for the service being open with `curl`.